src/style.css

```diff
@@ -56,11 +56,20 @@ body {
 }
 
 #map {
-  height: 780px;
+  height: 450px;
   width: 100vw;
+  margin: auto;
 }
-
-#flight-list {
+.mt-50{
+  margin-top: 50px
+}
+.ml-50{
+  margin-left: 50px;
+}
+.h-12 {
```

Comment:
this will conflict with tailwind as a class as h-12 in tailwind is 3 rem and you have it as a custom value as 3rem

Reply:
Removed possible conflicts, only using tailwind.